### PR TITLE
[APT-1575] Add captions to our images

### PR DESCRIPTION
**Pull request by ebeneliason**
- Captions are now displayed beneath images using an _emphasis_ convention,
  duplicating the value specified as the alt text for the image itself
- Both images and captions are now centered on the page
- The image lightbox background is now black in the dark theme

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -69,6 +69,26 @@ main {
...
 }
 
+/* IMAGES & CAPTIONS */
+
+p > img {
+  display: block;
+  margin: auto;
+}
+
+img + em {
```
Comment (eak12913) [inline src/css/custom.css:79]: I wasn't familiar with this selector - what does it mean to have `img + em` is it: 
```
<em><img /> </em>
```?

**Review by eak12913 — APPROVED** [review]
Looks great!

File: src/css/custom.css (hunk 1 above)
Comment (oredavids) [reply src/css/custom.css:79]: [Adjacent sibling combinator](https://css-tricks.com/child-and-sibling-selectors/#adjacent-sibling-combinator). 👍

**Review by oredavids — APPROVED** [review]
LGTM.

**Comment by ebeneliason** [comment]
I'm closing this in favor of #125, which achieves the same result with a plugin to avoid the redundant text in the source file.

**Pull request closed without merging**